Improve the qcomm aot part docs

cccclai opened the pull request:
Summary: the schema file copy is an actual AOT step. It's a workaround but it has been here a while, probably easier to have it as part of the build script instead of manually doing it everytime

Differential Revision: D63878765

shewu-quic commented on docs/source/build-run-qualcomm-ai-engine-direct-backend.md:
```diff
@@ -147,6 +147,10 @@ cmake --build $PWD --target "PyQnnManagerAdaptor" "PyQnnWrapperAdaptor" -j$(npro
 cp -f backends/qualcomm/PyQnnWrapperAdaptor.cpython-310-x86_64-linux-gnu.so $EXECUTORCH_ROOT/backends/qualcomm/python
+
+# Workaround for fbs files in exir/_serialize
+cp schema/program.fbs exir/_serialize/program.fbs
```
Because base dir is build-x86 here.
I think that it could add `$EXECUTORCH_ROOT` here.
```
cp $EXECUTORCH_ROOT/schema/program.fbs $EXECUTORCH_ROOT/exir/_serialize/program.fbs
cp $EXECUTORCH_ROOT/schema/scalar_type.fbs $EXECUTORCH_ROOT/exir/_serialize/scalar_type.fbs
```

shewu-quic commented on backends/qualcomm/scripts/build.sh:
```diff
@@ -135,6 +135,8 @@ if [ "$BUILD_X86_64" = true ]; then
 
     rm -f $PRJ_ROOT/backends/qualcomm/python/*
     cp -fv $BUILD_ROOT/backends/qualcomm/Py* "$PRJ_ROOT/backends/qualcomm/python"
+    cp  "$PRJ_ROOT/schema/program.fbs" "$PRJ_ROOT/exir/_serialize/program.fbs"
```
-fv

shewu-quic reviewed (approved):
LGTM

chiwwang reviewed (commented):
I need to delegate to @haowhsu-quic , @shewu-quic . Please help. Thanks!

haowhsu-quic reviewed (approved):
Thank you for helping this.

cccclai committed:
Improve the qcomm aot part docs (#5868)

Summary:

the schema file copy is an actual AOT step. It's a workaround but it has been here a while, probably easier to have it as part of the build script instead of manually doing it everytime

Differential Revision: D63878765

cccclai commented:
@pytorchbot cherry-pick --onto release/0.4 -c docs
